export class RecursoNaoEncotradoError extends Error {
  constructor() {
    super('Recurso n√£o encontrado')
  }
}
